op-service/dial/rollup_sync.go

```diff
@@ -0,0 +1,37 @@
+package dial
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+)
+
+func WaitRollupSync(ctx context.Context, lgr log.Logger,
```

Comment:
nit: Splitting the function args like this is a bit weird. It's a little long for a single line or it could be split with one arg per line.

IDK if the setup is worth it, but `lgr, rollup, & pollDuration` could all be init params on something that had a `Wait(ctx,target)` method

Reply:
+1 for "one arg per line" when the signature is too long.

Reply:
Split the args up into individual lines.

Yeah there could be a `SyncStatusWaiter` with a `Wait` function, but since all use cases of this that I envision only ever call this once, I'd rather just have a single function. I suggest for this iteration we leave it with the function, and if we ever see a case for reuse, or splitting up the initialization from the waiting, we create that `SyncStatusWaiter`.